metasmoke.py

```diff
@@ -62,18 +154,18 @@ def init_websocket():
                     a = GlobalVars.metasmoke_ws.recv()
                     try:
                         data = json.loads(a)
-                        GlobalVars.metasmoke_last_ping_time = datetime.utcnow()
```

Comment:
Is there really a use case of metasmoke_last_ping_time? Searching through the code base, this field does not appear to be read. Hence there is no calls to `get_last_ping()` Anyways I implemented a getter/setter to replace that. If there is no use case, maybe we can consider to completely get rid of that field, and also my `set_last_ping()` and `get_last_ping()`.